SparkPost/matchbox
# UX-363 Document InlineCode component

### What Changed

- Documents `InlineCode` Component

### How To Test or Verify

- `cd site && npm start`
- Verify `InlineCode` is documented correctly [http://localhost:8001/components/inlinecode](http://localhost:8001/components/inlinecode)

### PR Checklist
